docs/coding_style/accessibility.md

```diff
@@ -13,6 +13,51 @@ Other parts of web accessibility involve writing the interview in a way that the
 
+## Accessibility testing tools
+
+Testing your interview with assistive technology and accessibility tools is essential to ensure it works for all users. Here are some popular tools to help you test and validate the accessibility of your interviews:
+
+### Screen readers
+
+Screen readers are assistive technologies that read content aloud for users who are blind or have low vision. Testing with screen readers helps ensure your interview is usable for these users.
+
+**Free screen readers:**
+* **[NVDA (NonVisual Desktop Access)](https://www.nvaccess.org/)** - Free, open-source screen reader for Windows. Widely used and regularly updated.
+* **[VoiceOver](https://www.apple.com/accessibility/vision/)** - Built into macOS and iOS devices. Activate with Command+F5 on Mac.
+* **[Orca](https://help.gnome.org/users/orca/stable/)** - Open-source screen reader for Linux systems.
+* **[ChromeVox](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn)** - Chrome extension screen reader, useful for quick testing.
+
+**Commercial screen readers:**
+* **[JAWS (Job Access With Speech)](https://www.freedomscientific.com/products/software/jaws/)** - Popular commercial screen reader for Windows with advanced features.
+* **[Dragon NaturallySpeaking](https://www.nuance.com/dragon.html)** - Voice recognition software that can be used alongside screen readers.
+
+### Browser accessibility checkers
+
+These tools can automatically scan your interview pages for accessibility issues:
+
+* **[WAVE Web Accessibility Evaluator](https://wave.webaim.org/extension/)** - Browser extension that highlights accessibility issues directly on the page.
+* **[axe DevTools](https://www.deque.com/axe/devtools/)** - Browser extension for Chrome, Firefox, and Edge that integrates with developer tools.
+* **[Lighthouse](https://developer.chrome.com/docs/lighthouse/)** - Built into Chrome DevTools, includes accessibility auditing alongside performance testing.
+* **[Accessibility Insights](https://accessibilityinsights.io/)** - Microsoft's accessibility testing tools for web and Windows applications.
+
+### Manual testing tools
+
+* **[Colour Contrast Analyser](https://www.tpgi.com/color-contrast-checker/)** - Desktop application for testing color contrast ratios.
+* **[Accessibility bookmarklets](https://accessibility-bookmarklets.org/)** - Collection of browser bookmarklets for quick accessibility checks.
+* **[Web Accessibility Toolbar](https://www.tpgi.com/web-accessibility-toolbar/)** - Internet Explorer and Firefox toolbar with accessibility testing features.
+
+### Mobile accessibility testing
+
+* **[TalkBack](https://support.google.com/accessibility/android/answer/6283677)** - Android's built-in screen reader.
+* **[Switch Access](https://support.google.com/accessibility/android/answer/6122836)** - Android feature for users with motor disabilities.
+* **[Voice Control](https://support.apple.com/en-us/HT210417)** - iOS voice navigation feature.
```

Comment:
Same with Dragon, I don't think that Voice Control is going to really help people with testing accessibility: if it gets picked up with Voice Over, it'll work with Voice Control. I'd probably link Voice Over down here again, just to emphasize that it's the same tool, both on Macs and iPhones.